packages/components/src/components/nav-item/nav-item.styles.ts

```diff
@@ -253,7 +253,7 @@ export default css`
   }
 
   .nav-item--horizontal:hover .current-indicator--visible,
-  .nav-item--horizontal:focus .current-indicator--visible {
```

Comment:
Because of the commit message this change won't be promoted in the changelog. This should be a separate PR. (Sorry.)

Reply:
Its now removed and will be handled in #486 instead.